akd/src/append_only_zks.rs

```diff
@@ -725,6 +680,37 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    #[allow(dead_code)]
```

Comment:
nit: Do we need the dead code attribute? is it not in a `#[cfg(test)]` block? If not can we move them to a

```
#[cfg(test)]
mod tests {

...
}
```